app/controllers/events/view/sessions/list.js

```diff
@@ -1,218 +1,243 @@
 import Controller from '@ember/controller';
+import { action } from '@ember/object';
 import { mapBy } from '@ember/object/computed';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
 
-export default Controller.extend({
-  ratedSessions: mapBy('model.feedbacks', 'session.id'),
+export default class extends Controller.extend(EmberTableControllerMixin) {
+  ratedSessions= mapBy('model.feedbacks', 'session.id');
```

Comment:
Use decorator

Reply:
Do we have to use this like other computed properties?
```js
@computed()
get ratedSessions() {
    return mapBy('model.feedbacks', 'session.id');
}
```

Reply:
No ...

```js
@mapBy('model.feedbacks', 'session.id')
ratedSessions
```

@shreyanshdwivedi read the docs please -> http://api.emberjs.com/ember/3.10/functions/@ember%2Fobject%2Fcomputed/mapBy